### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by MaliJetal**
The code having good random values which give us the pretty recognition that noise is actually reducing about to zero.

### CODE REVIEW

File: kalman.py [hunk 1]
```diff
@@ -1,53 +1,35 @@
-"""
-A gross oversimplification of the the kalman filter
-For each timestep:
-current estimation = Gain * Measured + (1 - Gain) * previousestimation
-
-The only unknown is the gain, this is continually revised using new
-info and something from the previous state Kalman filter finds the
-most optimum averaging factor for each consequent state. Also somehow
-remembers a little bit about the past states
-a very good dummy intro:
```
Comment (squeakus) [inline kalman.py:10]: could you add this link back in, I found it useful for understanding the code

**Review by squeakus — CHANGES_REQUESTED** [review]
Just one comment and then i'll approve

File: kalman.py (hunk 1 above)
Comment (MaliJetal) [reply kalman.py:10]: I have uploaded the 2 .py files. I didn't get you which link do u want?

Comment (squeakus) [reply kalman.py:10]: http://bilgin.esme.org/BitsBytes/KalmanFilterforDummies.aspx

Comment (MaliJetal) [reply kalman.py:10]: Yeah Sure

**Comment by MaliJetal** [comment]
Yeah Sure

File: kalman.py (hunk 1 above)
Comment (MaliJetal) [reply kalman.py:10]: i added the link. Do check.

Comment (MaliJetal) [reply kalman.py:10]: [randomnum.zip](https://github.com/squeakus/kalman/files/2700760/randomnum.zip)

**Comment by squeakus** [comment]
Done! Thanks for contributing

File: kalman.py (hunk 1 above)
Comment (MaliJetal) [reply kalman.py:10]: Thankew so much by pushing it up.

**Pull request merged**